src/main/java/BasketInformations.java

```diff
@@ -0,0 +1,98 @@
+package com.company;
+
+import java.util.HashMap;
+
+/**
+ * This class is a basket
+ */
+public class BasketInformations {
+
+	// The product of the basket
+	static HashMap<String, Integer> map = new HashMap<String, Integer>();
+
+	static String[] localProducts = new String[]{"123", "222", "44", "657"};
```

Comment:
```suggestion
	private static final List<String> localProducts = Arrays.asList(new String[]{"123", "222", "44", "657"});
```
Same as `map` localProducts is a class variable (`static') rather than an instance one, maybe it is intended to be, I would suggest we change its modifier to prevent reassigning.

- type: Since we intend to search inside this array, I suggest we change it to one of the Collections API implementation, like `ArrayList`